src/Client.php

```diff
@@ -335,7 +335,32 @@ public function customerInfo($customerId)
 
 		return $this->processRequest(Message\Request::customerInfo($data));
 	}
+	
+	/**
+	 * @return Message\Response
+	 */
+	public function echoGET()
+	{
+		$data = [
+			'merchantId' => $this->config->getMerchantId(),
+			'dttm' => $this->formatDatetime(),
+		];
+
+		return $this->processRequest(Message\Request::echoGET($data));
+	}
 
+	/**
+	 * @return Message\Response
+	 */
+	public function echoPOST()
```

Comment:
These two methods are basically the same, I'd rather have the HTTP method parametrized, i.e. single `function echo($method = Request::GET)`